src/tcp.rs

```diff
@@ -421,4 +427,36 @@ mod tests {
         assert_eq!(deserialized, header);
         assert_eq!(re_deserialized, header);
     }
+    #[test]
+    fn try_clone(){
+        let mut close = Arc::new(Mutex::new(()));
+        thread::scope(|s| {
+            let mut close_clone = close.clone();
+            let jh = s.spawn(move ||{
+                let listener = TcpListener::bind("127.0.0.1:34254").unwrap();
+                listener.accept().and_then(|(tcp,addr)| {
```

Comment:
Substitute the unused parameters `tcp` and `addr` with `_` or prefixed (`_tcp`) to silence rustc warnings.